packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -0,0 +1,164 @@
+import 'ng-slide-down';
+import '@ovh-ux/ng-ovh-actions-menu';
+import 'script-loader!matchmedia-ng/matchmedia-ng.js';
+import 'angular-aria';
+import 'script-loader!chart.js/dist/Chart.min.js';
+import 'script-loader!angular-chart.js/dist/angular-chart.min.js';
+import 'ovh-angular-responsive-tabs';
+import 'ckeditor';
+import 'script-loader!messenger/build/js/messenger.min.js';
+import 'script-loader!filesize/lib/filesize.js';
+
+// Ckeditor 4.x
+import 'ng-ckeditor';
+
+import './app.less';
+import './css/source.scss';
+/* eslint-enable import/no-webpack-loader-syntax, import/no-unresolved, import/extensions */
+
+import moduleName from './app';
+import './app.routes';
+
+import './account/contacts/service/edit/edit.module';
+import './account/otrs/otrs-controllers.module';
+import './account/otrs/otrs-directives.module';
+import './account/otrs/otrs-filters.module';
+import './account/otrs/otrs-services.module';
+import './account/user/advanced/advanced.module';
+import './account/user/dashboard/user-dahboard.module';
+import './account/user/newAccountForm/new-account-form.module';
+import './account/user/support-level/support-level.module';
+import './account/user/user.module';
+import './billing/autoRenew/actions/activation/activation.module';
+import './billing/autoRenew/actions/cancel-resiliation/cancel-resiliation.module';
+import './billing/autoRenew/actions/debtBeforePaying/debtBeforePaying.module';
+import './billing/autoRenew/actions/delete/delete.module';
+import './billing/autoRenew/actions/exchange/exchange-renew.module';
+import './billing/autoRenew/actions/terminate-enterprise-cloud-database/terminate-enterprise-cloud-database.module';
+import './billing/autoRenew/actions/terminate-webcoach/terminate-webcoach.module';
+import './billing/autoRenew/actions/terminate/terminate.module';
+import './billing/autoRenew/actions/terminateEmail/email.module';
+import './billing/autoRenew/actions/terminateHostingWeb/hosting-web.module';
+import './billing/autoRenew/actions/terminatePrivateDatabase/private-database.module';
+import './billing/autoRenew/actions/update/automatic/automatic.module';
+import './billing/autoRenew/actions/update/form/form.module';
+import './billing/autoRenew/actions/update/manualPayment/manualPayment.module';
+import './billing/autoRenew/actions/update/noPaymentMean/noPaymentMean.module';
+import './billing/autoRenew/actions/update/update.module';
+import './billing/autoRenew/actions/warnNicBilling/warnNicBilling.module';
+import './billing/autoRenew/actions/warnPendingDebt/pending-debt.module';
+import './billing/autoRenew/agreements/details/details.module';
+import './billing/autoRenew/agreements/user-agreements.module';
+import './billing/autoRenew/autorenew.module';
+import './billing/autoRenew/bulk/bulk.module';
+import './billing/autoRenew/disable-domains-bulk/disable-domains-bulk.module';
+import './billing/autoRenew/disable/disable.module';
```

Comment:
Why do we have to import all billing/autoRenew module ? Aren't they already imported by their parent ?

Reply:
I just move `glob` from webpack config to javascript `import`. So it works like before.
I put them in `bundle` as it was previously named bundle in webpack. 
We have to clean theses files and avoid to import a list of js files.